src/clr-angular/utils/popover/providers/popover-toggle.service.ts

```diff
@@ -59,4 +60,14 @@ export class ClrPopoverToggleService {
     this.openEvent = event;
     this.open = !this.open;
   }
+
+  private _popoverAligned: Subject<HTMLElement> = new Subject();
+
+  public get popoverAligned(): Observable<HTMLElement> {
```

Comment:
This needs a test that this public property return an observable if that type.

Reply:
Done.